tubular/nominal.py

```diff
@@ -1819,6 +1875,7 @@ def _warn_missing_levels(
 
         return missing_levels
 
+    @block_from_json
```

Comment:
I know we wanted to block fitting a transformer reconstructed from JSON (hence the @block_from_json on fit method). Just wanted to double check if we indeed want to block get_feature_names_out from working on the transformer reconstructed from JSON as well?